src/components/header/header.scss

```diff
@@ -0,0 +1,60 @@
+$primary-color: rgb(27, 41, 48);
+$logo-color: rgb(246, 205, 77);
+$border-color: rgb(150, 150, 150);
+$social-icon-size: 4rem;
+
+#header {
+  background: $primary-color;
+  display: flex;
+  height: 10rem;
```

Comment:
Is there any reason for a fixed height here? If it's a matter of containing the size of elements, I'd propose setting a `width` on the element and letting height be it's natural height.